discord/user.go

```diff
@@ -129,31 +129,37 @@ const (
 )
 
 type Activity struct {
-	Name string `json:"name"`
-	URL  URL    `json:"url,omitempty"`
-
-	Type ActivityType `json:"type"`
-
 	// User only
-
-	Instance bool          `json:"instance,omitempty"`
-	Flags    ActivityFlags `json:"flags,omitempty"`
-
-	CreatedAt  UnixTimestamp      `json:"created_at,omitempty"`
 	Timestamps *ActivityTimestamp `json:"timestamps,omitempty"`
-
-	AppID   AppID  `json:"application_id,omitempty"`
-	Details string `json:"details,omitempty"`
-	State   string `json:"state,omitempty"` // party status
-	Emoji   *Emoji `json:"emoji,omitempty"`
-
-	Party   *ActivityParty   `json:"party,omitempty"`
-	Assets  *ActivityAssets  `json:"assets,omitempty"`
+	// User only
+	Emoji *Emoji `json:"emoji,omitempty"`
+	// User only
 	Secrets *ActivitySecrets `json:"secrets,omitempty"`
-
+	// User only
+	Assets *ActivityAssets `json:"assets,omitempty"`
+	// User only
+	Party *ActivityParty `json:"party,omitempty"`
 	// Undocumented fields
-	SyncID    string `json:"sync_id,omitempty"`
+	// User only
+	SyncID string `json:"sync_id,omitempty"`
+	// User only
+	State string `json:"state,omitempty"`
+	URL   URL    `json:"url,omitempty"`
+	// User only
+	Details string `json:"details,omitempty"`
+	Name    string `json:"name"`
+	// Undocumented fields
+	// User only
 	SessionID string `json:"session_id,omitempty"`
+	// User only
+	AppID AppID `json:"application_id,omitempty"`
+	// User only
+	CreatedAt UnixTimestamp `json:"created_at,omitempty"`
+	// User only
+	Flags ActivityFlags `json:"flags,omitempty"`
+	// User only
+	Instance bool         `json:"instance,omitempty"`
+	Type     ActivityType `json:"type"` // Undocumented fields
```

Comment:
Remove the EOL comment and add a doc.

```suggestion
	// Type is the activity's type.
	Type     ActivityType `json:"type"`
```